src/ui/app/components/transaction.jsx

```diff
@@ -495,9 +495,43 @@ const getTimestamp = (date) => {
 };
 
 const getAddressCredentials = (address) => {
-  const cmlAddress = Loader.Cardano.Address.from_bech32(address);
-  return [cmlAddress.payment_cred()?.to_cbor_hex(), cmlAddress.staking_cred()?.to_cbor_hex()];
-}
+  try {
+    const cmlAddress = Loader.Cardano.Address.from_bech32(address);
+    return [
+      cmlAddress.payment_cred()?.to_cbor_hex() || null,
+      cmlAddress.staking_cred()?.to_cbor_hex() || null,
+    ];
+  } catch (error) {
+    try {
+      // try casting as byron address
+      const cmlAddress = new Loader.Cardano.ByronAddress(address);
```

Comment:
Not sure if constructor does the same (it might). Just FIY:
```suggestion
      const cmlAddress = Loader.Cardano.ByronAddress.from_base58(address);
```